src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,73 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String SUPPLY_WORD = "supply";
+    private static final String BUY_WORD = "buy";
+    private static final String RESULT_WORD = "result";
+    private static final int NAME_INDEX = 0;
+    private static final int NUMBER_INDEX = 1;
+    private static final String GET_SUM_SUPPLY = "getSupply";
+    private static final String GET_SUM_BUY = "getBuy";
+
     public void getStatistic(String fromFileName, String toFileName) {
```

Comment:
this method should contain three calls of other methods: readFile() - returns content, generateReport() -(you pass contents here), writeToFile()-(you pass report and filename here ), this will be a good code decomposition

Reply:
Not fixed